pom.xml

```diff
@@ -78,6 +78,12 @@
             <artifactId>httpclient</artifactId>
             <version>4.5.13</version>
         </dependency>
+        <!-- force commons-codec version to address CVE CCMSG-835 -->
```

Comment:
let's mention that this dependency is a transitive dependency of `httpclient` above. Unfortunately there's no recent enough version of `httpclient` that would allow us to use and upgrade the codec without having to explicitly list it as a dependency. Thus, your fix makes sense (as long as it doesn't break the `httpclient`) but we should consider for removal once `httpclient` gets upgraded in the future. So, a comment would help here.